riscof-plugin/sail_cSim/sail_isa.yaml

```diff
@@ -0,0 +1,38 @@
+hart_ids: [0]
```

Comment:
Sail is both RV32 and RV64, so having just a sail_isa.yaml that's RV32 is wrong. Moreover, many aspects of the model are configurable (and it will eventually become even more so), so having this YAML file be static does not seem right either (e.g. RVC can be turned off).